### PR TITLE
Add sntrup{953,1013,1277} and ntrulpr{953,1013,1277}

### LINKED ISSUE #407: Add sntrup{953,1013,1277} and ntrulpr{953,1013,1277}
pqm4 received a PR for the larger round 3 parameter sets (https://github.com/mupq/pqm4/pull/202). 
It would be nice if we could also pull them into PQClean from supercop.

### PR DESCRIPTION
**Pull request by mkannwischer**
This PR adds the new round 3 parameter sets of NTRU Prime. 
Implementations come from supercop ([20210604](https://bench.cr.yp.to/supercop/supercop-20210604.tar.xz)) packaged via @jschanck's scripts. Modifications are available at https://github.com/mkannwischer/package-pqclean/tree/main/ntruprime. 

Resolves #407.

### CODE REVIEW

File: crypto_kem/ntrulpr1013/avx2/api.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_ALGNAME "ntrulpr1013"
+
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_SECRETKEYBYTES 1773
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_PUBLICKEYBYTES 1455
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_CIPHERTEXTBYTES 1583
+#define PQCLEAN_NTRULPR1013_AVX2_CRYPTO_BYTES 32
+
+int PQCLEAN_NTRULPR1013_AVX2_crypto_kem_keypair(unsigned char *pk, unsigned char *sk);
+int PQCLEAN_NTRULPR1013_AVX2_crypto_kem_enc(unsigned char *c, unsigned char *k, const unsigned char *pk);
+int PQCLEAN_NTRULPR1013_AVX2_crypto_kem_dec(unsigned char *k, const unsigned char *c, const unsigned char *sk);
```
Comment (thomwiggers) [inline crypto_kem/ntrulpr1013/avx2/api.h:15]: Did we not specify these should be `uint8_t`?

**Review by thomwiggers — DISMISSED** [review]
For the most part I think it looks good.

File: crypto_kem/ntrulpr1013/avx2/api.h (hunk 1 above)
Comment (mkannwischer) [reply crypto_kem/ntrulpr1013/avx2/api.h]: Indeed. I changed it. Thanks

**Comment by thomwiggers** [comment]
Seems there's one minor issue in sntru remaining.

**Review by thomwiggers — APPROVED** [review]

**Pull request merged**